src/components/BookingModal/BookingModalForm/index.js

```diff
@@ -91,44 +95,48 @@ const BookingModalForm = props => {
     <Fragment>
       {composeErrorMessage()}
       <Form formData={formData} formStatus={formStatus} actions={createCtas()}>
-        <Input
+        {!rejectionMessages ? <Input
```

Comment:
All of this conditional rendering could be handled better - it seems like this form's entire contents depend on `rejectionMessages` being false, so maybe wrap it in a single ternary.

Reply:
I don't think they can be blocked together with the logic as only some of the inputs depend independently upon only rejectionMessages. There are some others throughout the forms composition that depend upon cancelled event too and they don't render within a single block.